lib/emis/models/deployment_location.rb

```diff
@@ -13,6 +14,8 @@ class DeploymentLocation
       attribute :termination_reason_code, String
```

Comment:
Do we want to document these Virtus attributes? I have been, copying the way @kreek did it [here](https://github.com/department-of-veterans-affairs/vets-api/pull/2899/files).

Reply:
i'll add it in another pr